test/libsolidity/syntaxTests/largeTypes/oversized_contract_inheritance.sol

```diff
@@ -8,5 +8,3 @@ contract D is C {
 }
 // ----
 // TypeError 7676: (95-134): Contract too large for storage.
```

Comment:
Does this try to say contract is usage too much storagE?

Reply:
We sum up state variable sizes
```
for (ContractDefinition const* contract: boost::adaptors::reverse(_contract.annotation().linearizedBaseContracts))
	for (VariableDeclaration const* variable: contract->stateVariables())
		if (!(variable->isConstant() || variable->immutable()))
		{
			size += variable->annotation().type->storageSizeUpperBound();
			if (size >= bigint(1) << 256)
			{
				m_errorReporter.typeError(7676_error, _contract.location(), "Contract too large for storage.");
				break;
			}
		}
```

The message is similar to "Type too large for storage".

Reply:
I'd reconsider the message as it is not entirely clear, to me at least.

Reply:
Would "Contract requires to much storage" be OK?

Reply:
Sounds good to me. Perhaps `Contract allows addressing too much storage with potential for conflicts.`

Reply:
Note that this is not about potential for conflicts, it is about the storage slot allocator function overflowing.

Reply:
The error  "Contract requires too much storage" is part of PR #9818 now.